start_async_worker.py

```diff
@@ -0,0 +1,21 @@
+from logging.config import dictConfig as load_dict_config
+
+import sentry_sdk
+from sentry_sdk.integrations.rq import RqIntegration
+
+from busy_beaver.app import create_app
+from busy_beaver.config import IN_PRODUCTION, LOGGING_CONFIG, SENTRY_DSN
+from busy_beaver.extensions import rq
+
+# observability
+LOGGING_CONFIG["handlers"]["datadog_file"]["filename"] = "logs/busy_beaver_worker.json"
```

Comment:
Good enough for now. This will be changed as the project matures and we have more workers.

Will start grabbing Docker logs.